storybookjs/storybook
# feat(ui, preview): add loader to canvas's preview

Issue: [#9127](https://github.com/storybookjs/storybook/issues/9127)

## What I did

- Added a **Loader** element to the **Preview's canvas tab** during the loading time of the stories to give a visual feedback to the visitors on mobile.

## How to test

To test this feature, simply launch any storybook and check that a loader element is displayed on the canvas tab once the page has loaded and that the stories are being fetched, for both Desktop & Mobile modes.
